articles/fleet-software-attestation.md

```diff
@@ -22,13 +22,17 @@ Verify the [fleetctl binary](https://github.com/fleetdm/fleet/releases) (CLI):
 gh attestation verify --owner fleetdm fleetdm /path/to/fleetctl
```

Comment:
While you are at it, it seems `fleetdm` here is twice.

@sgress454 Seems like a typo?